assets/locales/es-ES/translation.json

```diff
@@ -3,13 +3,13 @@
   "de": "Deutsch",
   "ru": "Ruso",
   "ja": "Japonés",
-  "zh-CN": "中文",
-  "ko": "한국어",
+  "zh-CN": "Chino",
```

Comment:
Perhaps we should keep the original language since a person who wants to change the language does not know other languages. They need their language to change language properly.
I'll update the root translation later